frame/staking/src/weights.rs

```diff
@@ -372,52 +367,54 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	fn new_era(v: u32, n: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 897_000
-			.saturating_add(Weight::from_ref_time(213_100_000 as u64).saturating_mul(v as u64))
-			// Standard Error: 45_000
-			.saturating_add(Weight::from_ref_time(31_123_000 as u64).saturating_mul(n as u64))
-			.saturating_add(T::DbWeight::get().reads(208 as u64))
-			.saturating_add(T::DbWeight::get().reads((5 as u64).saturating_mul(v as u64)))
+		Weight::from_ref_time(572_530_000 as u64)
+			// Standard Error: 3_166_542
+			.saturating_add(Weight::from_ref_time(101_354_358 as u64).saturating_mul(v as u64))
+			// Standard Error: 315_238
+			.saturating_add(Weight::from_ref_time(15_565_319 as u64).saturating_mul(n as u64))
+			.saturating_add(T::DbWeight::get().reads(261 as u64))
+			.saturating_add(T::DbWeight::get().reads((1 as u64).saturating_mul(v as u64)))
 			.saturating_add(T::DbWeight::get().reads((4 as u64).saturating_mul(n as u64)))
-			.saturating_add(T::DbWeight::get().writes(3 as u64))
+			.saturating_add(T::DbWeight::get().writes(6 as u64))
 			.saturating_add(T::DbWeight::get().writes((3 as u64).saturating_mul(v as u64)))
 	}
-	// Storage: BagsList CounterForListNodes (r:1 w:0)
+	// Storage: VoterBagsList CounterForListNodes (r:1 w:0)
 	// Storage: Staking SlashingSpans (r:21 w:0)
-	// Storage: BagsList ListBags (r:200 w:0)
-	// Storage: BagsList ListNodes (r:1500 w:0)
+	// Storage: VoterBagsList ListBags (r:200 w:0)
+	// Storage: VoterBagsList ListNodes (r:1500 w:0)
 	// Storage: Staking Nominators (r:1500 w:0)
 	// Storage: Staking Validators (r:500 w:0)
 	// Storage: Staking Bonded (r:1500 w:0)
 	// Storage: Staking Ledger (r:1500 w:0)
-	fn get_npos_voters(v: u32, n: u32, s: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 116_000
-			.saturating_add(Weight::from_ref_time(23_745_000 as u64).saturating_mul(v as u64))
-			// Standard Error: 116_000
-			.saturating_add(Weight::from_ref_time(22_497_000 as u64).saturating_mul(n as u64))
-			// Standard Error: 3_968_000
-			.saturating_add(Weight::from_ref_time(20_676_000 as u64).saturating_mul(s as u64))
-			.saturating_add(T::DbWeight::get().reads(202 as u64))
-			.saturating_add(T::DbWeight::get().reads((5 as u64).saturating_mul(v as u64)))
-			.saturating_add(T::DbWeight::get().reads((4 as u64).saturating_mul(n as u64)))
-			.saturating_add(T::DbWeight::get().reads((1 as u64).saturating_mul(s as u64)))
+	/// The range of component `v` is `[500, 1000]`.
+	/// The range of component `n` is `[500, 1000]`.
+	/// The range of component `s` is `[1, 20]`.
+	fn get_npos_voters(v: u32, n: u32, _s: u32, ) -> Weight {
+		Weight::from_ref_time(24_930_788_000 as u64)
+			// Standard Error: 266_386
+			.saturating_add(Weight::from_ref_time(6_687_552 as u64).saturating_mul(v as u64))
+			// Standard Error: 266_386
+			.saturating_add(Weight::from_ref_time(6_839_134 as u64).saturating_mul(n as u64))
+			.saturating_add(T::DbWeight::get().reads(6722 as u64))
```

Comment:
I am just seeing this now :sweat_smile:  
It is either a bug (fixed in https://github.com/paritytech/substrate/pull/12482) or expected.